frontend/packages/ux-editor/src/containers/DesignView/DesignView.tsx

```diff
@@ -22,7 +22,7 @@ import type { PageModel } from 'app-shared/types/api/dto/PageModel';
 import { DesignViewNavigation } from '../DesignViewNavigation';
 import { shouldDisplayFeature, FeatureFlag } from 'app-shared/utils/featureToggleUtils';
 import { PageGroupAccordion } from './PageGroupAccordion';
-import { useAddGroupMutation } from 'app-development/hooks/mutations';
+import { useAddGroupMutation } from '@altinn/ux-editor/hooks/mutations/useAddGroupMutation';
```

Comment:
I believe we decided to go for relative paths for package local imports, like:
```suggestion
import { useAddGroupMutation } from '../../hooks/mutations/useAddGroupMutation';
```